### DISCUSSION TITLE
[DONE ✅] Improve policy options ux in server settings

### DISCUSSION THREAD

**Discussion by britttttk** [btcpayserver/btcpayserver#2307] [Ideas / Feature Requests]
I find the current list of policies a little hard to read. I proposed in the chat to reword and reorder them. @dennisreimann also suggested grouping and adding doc links. This is my suggestion, also a few rewordings:

### Server Admin Policies

Existing User Settings
- Allow non-admins to use the internal lightning node in their stores
- Allow non-admins to create hot wallets for their stores
- Allow non-admins to import hot wallets for their stores

New User Settings
- Disable new user registration on the server
- Disable non-admins access to the user creation API endpoint
- Require a confirmation email for registering

Notification Settings
- Disable notifications from automatically showing (no websockets)
- Disable stores from using the server's email settings as backup

Maintenance Settings
- Check releases on GitHub and alert when new BTCPayServer version is available
- Discourage search engines from indexing this site

Currently it's this:
![image](https://user-images.githubusercontent.com/39231115/108806060-eb003600-755d-11eb-9398-3b43df58a5ca.png)

**Comment by Zaxounette** [comment]
In the string `Check releases on GitHub and alert when new BTCPayServer version is available`, the word `alert` could be replaced by `notify`, making the link between that setting and the `notifications` feature it uses more explicit ?

Other than that, good idea !

**Comment by dstrukt** [comment]
<img width="1047" alt="Screen Shot 2021-02-28 at 2 26 44 PM" src="https://user-images.githubusercontent.com/6250771/109435775-01304b00-79d1-11eb-81a7-4f5e58244d53.png">

Had mocked this out when I first started bringing over all the screens to Figma, same idea, but slightly different groupings, will give it another look over and see where we aligned.

Regardless, I love that we all seem to be on the same page!

Will get this cleaned up for @dennisreimann to implement.

Would be good to open a PR if there isn't one already -- should be a relatively easy patch.

**Reply by Zaxounette** [reply]
@dstrukt ,
#2308 has been merged a week ago.
That PR  is juste a re-ordering change to add readability, no UI/UX specific change.

We could close this discussion since it was more of an issue, and open another one specifically for the UI/UX part that your changes relate to.

**Reply by dstrukt** [reply]
Ahhh I see, no additional UI/UX proposal from me, but would definitely support a grouping for improved legibility.

**Reply by Zaxounette** [reply]
Grouping both for the next release could be a good idea yeah.
Denissssssssssssssssssssssssssssssssssssssssssss ! @dennisreimann

**Reply by britttttk** [reply]
I like the green checkboxes very nice. Although I think the groupings Dennis made were quite clear. I agree this should likely be closed.

**Reply by Eskyee** [reply]
Yes I liked @dennisreimann groupings

**Reply by britttttk** [reply]
I don't see how to close it. @pavlenex

**Reply by dstrukt** [reply]
Oh yeah, those colored checkmarks, etc.. will be addressed UI updates with Bootstrap changes (dennis and I working on already, so no need for extra ticket, etc..), but I've gone through the linked PR, and it looks like you guys covered everything. 

I just completely missed it - great work!

Nothing else from me on this one.

**Reply by Zaxounette** [reply]
You missed it probably because Github messes up things with discussions.
You can't tag or close discussions via PR's. Makes kinda sense, but still.

**Comment by Eskyee** [comment]
Been going thru most of the UX and it’s clear and simple... not found nothing at the moment!! Everything seems to be working fine on the testing server @britttttk will dig deeper tonight.. no problems so far... we love Colours that make it instinctively easy, and confident 🙌✨📸 great work in front of me ❤️✅

**Comment by NicolasDorier** [comment]
we can close this?

**Reply by pavlenex** [reply]
There's no really way to close discussions for now unfortunately, I believe it's something they're working on, either archiving or hiding from the main view. We can delete but I don't think that's good.

**Reply by ghost** [reply]
yes, fwiw maintainers can `lock` the conversation and prevent new comments being written.

**Reply by dennisreimann** [reply]
Isn't there a feature to turn discussions into issues?

**Reply by pavlenex** [reply]
Not yet, but issues -> discussions is possible.

**Reply by dennisreimann** [reply]
Ah ok, I must have misread that. Thanks!
